sdks/builds/ios.mk

```diff
@@ -113,30 +113,37 @@ endef
 ios_sysroot = -isysroot $(XCODE_DIR)/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS$(IOS_VERSION).sdk -miphoneos-version-min=$(IOS_VERSION_MIN)
 tvos_sysroot = -isysroot $(XCODE_DIR)/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS$(TVOS_VERSION).sdk 	-mtvos-version-min=$(TVOS_VERSION_MIN)
 watchos_sysroot = -isysroot $(XCODE_DIR)/Platforms/WatchOS.platform/Developer/SDKs/WatchOS$(WATCHOS_VERSION).sdk -mwatchos-version-min=$(WATCHOS_VERSION_MIN)
+watchos5_sysroot = -isysroot $(XCODE_DIR)/Platforms/WatchOS.platform/Developer/SDKs/WatchOS$(WATCHOS5_VERSION).sdk -mwatchos-version-min=$(WATCHOS5_VERSION_MIN)
 
 # explicitly disable dtrace, since it requires inline assembly, which is disabled on AppleTV (and mono's configure.ac doesn't know that (yet at least))
 ios-targettv_CONFIGURE_FLAGS = 	--enable-dtrace=no --enable-llvm-runtime --with-bitcode=yes --with-monotouch-tv
 ios-targetwatch_CONFIGURE_FLAGS = --enable-cooperative-suspend --enable-llvm-runtime --with-bitcode=yes --with-monotouch-watch
+ios-targetwatch64_32_CONFIGURE_FLAGS = --enable-cooperative-suspend --with-bitcode=yes --with-monotouch-watch
 
 ios-target32_SYSROOT = $(ios_sysroot)
 ios-target32s_SYSROOT = $(ios_sysroot)
 ios-target64_SYSROOT = $(ios_sysroot)
 ios-targettv_SYSROOT = $(tvos_sysroot)
 ios-targetwatch_SYSROOT = $(watchos_sysroot)
+ios-targetwatch64_32_SYSROOT = $(watchos5_sysroot)
 
 ios-target32_CPPFLAGS = -DHOST_IOS
 ios-target32s_CPPFLAGS = -DHOST_IOS
 ios-target64_CPPFLAGS = -DHOST_IOS
 ios-targettv_CPPFLAGS = -DHOST_APPLETVOS -DTARGET_APPLETVOS
 ios-targetwatch_CPPFLAGS = -DHOST_IOS -DHOST_WATCHOS
+ios-targetwatch64_32_CPPFLAGS = -DHOST_IOS -DHOST_WATCHOS
```

Comment:
Question related to the surrounding code instead of this PR: why does `targettv_CPPFLAGS` not define HOST_IOS when both the iOS and watchOS versions do? And why doesn't the iOS and watchOS variables define a TARGET_* variable when the tvOS one does?

Reply:
good question, I opened an issue: https://github.com/mono/mono/issues/11697